app/views/GoodsDescriptionView.scala.html

```diff
@@ -30,16 +32,24 @@
 
     @formHelper(action = routes.GoodsDescriptionController.onSubmit(mode)) {
 
+        <h1 class="govuk-heading-xl">@messages("goodsDescription.h1")</h1>
+        <p class="govuk-body">@messages("goodsDescription.p1")</p>
+        <p class="govuk-body">@messages("goodsDescription.p2")</p>
+
         @if(form.errors.nonEmpty) {
             @govukErrorSummary(ErrorSummaryViewModel(form))
         }
 
-        @govukInput(
-            InputViewModel(
+        @textArea(
```

Comment:
There's <textarea class="govuk-textarea" , not sure if that suffices the need though ):

Reply:
If we can use a HMRC pre defined model, this should be okay :)